patches/third_party-blink-renderer-core-frame-local_frame.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/frame/local_frame.cc b/third_party/blink/renderer/core/frame/local_frame.cc
+index 90cf3971e573aaec7a85ccf4b6fff9f931924bdc..af75df20198dbba4429d6317441690d7fde89de6 100644
+--- a/third_party/blink/renderer/core/frame/local_frame.cc
++++ b/third_party/blink/renderer/core/frame/local_frame.cc
+@@ -1565,6 +1565,7 @@ LocalFrame::LocalFrame(LocalFrameClient* client,
+   idleness_detector_ = MakeGarbageCollected<IdlenessDetector>(this, clock);
+   attribution_src_loader_ = MakeGarbageCollected<AttributionSrcLoader>(this);
+   inspector_task_runner_->InitIsolate(isolate);
++  BRAVE_LOCAL_FRAME_CONSTRUCTOR
```

Comment:
how early does this need to be called? Can we do `#define FrameAttachedToParent(FRAME, AD_SCRIPT_ON_STACK) ....;`?

Reply:
> how early does this need to be called? Can we do `#define FrameAttachedToParent(FRAME, AD_SCRIPT_ON_STACK) ....;`?

yep, it's fine to call it at this point, removed the patch. thanks!